acceptance/samples/policy-input-golden-container.json

```diff
@@ -887,7 +887,8 @@
                     ],
                     "IMAGE_EXPIRES_AFTER": "",
                     "STORAGE_DRIVER": "vfs",
-                    "TLSVERIFY": "true"
+                    "TLSVERIFY": "true",
+                    "HERMETIC": "true"
```

Comment:
This makes sense since the task has the param, [see here](https://github.com/konflux-ci/build-definitions/blob/5d128a5e3f502e1f9da13224c09dd763ace303fe/task/buildah-oci-ta/0.4/buildah-oci-ta.yaml#L81-L83).